library/kotlin/src/io/envoyproxy/envoymobile/filters/ResponseFilter.kt

```diff
@@ -50,6 +50,15 @@ interface ResponseFilter : Filter {
    */
   fun onResponseTrailers(trailers: ResponseTrailers): FilterTrailersStatus<ResponseTrailers>
 
+  /**
+   * Invoked explicitly in response to an asynchronous resume() callback when filter
+   * iteration has been stopped.
```

Comment:
- These functions are currently called `continueRequest()` / `continueResponse()`; I think they need to be updated in this PR on the callback types
- Can we add documentation to the callbacks indicating that this function will be called on the filter's execution thread as an immediate result of calling the callback?
- It'd be nice to add an example use case to these descriptions. Personally I think we should be as clear/verbose as possible on these interface descriptions

Reply:
- Updated.
- Sure.
- Good idea.